docs/user-guide.md

```diff
@@ -336,6 +336,46 @@ In the file menu, you can choose to unshare it:
 1. In the Manage attachments dialog, point to a file and select it. You can select several files.
 2. Click the **Delete** icon below the files tree.
 
+### Publish
+
+You can publish conversations to make them available for the target audience. You can also unpublish them to withdraw from public use.
+
+**Note**: to publish a shared conversation, [duplicate](#duplicate) it and then publish. 
+
+**To publish a conversation**: 
+
+1. Click **Publish** in the conversation menu. 
+
+![](./img/publish-conversation.png)
+
+2. In the pop-up window:
+   * Enter **publication request name** in the upper area.
+   * In **Publish to**, you can select the target audience. You can also create a dedicated folder to publish your conversation into.
+   * In **Allow access..**, specify access rules. For example Role-Equals-Admin. **Note**: the available roles are configured for each organization individually. Contact your support to find out the rules are applicable in your organization.
+   * In **Conversations**, you can choose what conversations you want to publish (if you have selected a folder with several conversations). If conversations include attachments, you can also select them in the **Files** section.
```

Comment:
Didn't understand the phrase (if you have selected a folder with several conversations)
Do you mean that we can publish the whole folder?

Reply:
Yes

Reply:
Yes

Reply:
Yes